cmd/bpf2go/main_test.go

```diff
@@ -90,6 +90,8 @@ func main() {
 			goBuild.Env = append(os.Environ(),
 				"GOOS=linux",
 				"GOARCH="+arch,
+				"GOPROXY=off",
```

Comment:
The cache isn't populated, since we're cross compiling for another arch. vimto currently doesn't have network set up, so we don't really need to do anything extra. GOPROXY=off avoids the build failing since the binary can't access proxy.golang.org.